### PR TITLE
add active_backup_for_business functionality

### PR DESCRIPTION
**Pull request by raph2i**
Hey!

I'm about to develop a prometheus exporter, which will use this api.
Unfortunately, active backup for business vm-backup is missing, so i coded it ;)

It has been tested with a DS1815+ and 2 configured hypervisors with about 10 vms.

### CODE REVIEW

**Comment by N4S4** [comment]
Hi,

I tested as well your integration and works flawlessly, although I need you to do some little modification to put your PR in line with the master branch, you will find comments in the code

thank you for contributing

File: synology_api/active_backup.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
+from . import auth as syn
+
+import time
+
+class ActiveBackupBusiness:
+    def __init__(self, ip_address, port, username, password, secure=False):
```
Comment (N4S4) [inline synology_api/active_backup.py:6]: def __init__(self, ip_address, port, username, password, secure=False, cert_verify=False):

File: synology_api/active_backup.py [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+        self.session = syn.Authentication(ip_address, port, username, password, secure)
```
Comment (N4S4) [inline synology_api/active_backup.py:8]: self.session = syn.Authentication(ip_address, port, username, password, secure, cert_verify)

**Review by N4S4 — CHANGES_REQUESTED** [review]
please make this little changes and PR to master so I can merge

**Comment by raph2i** [comment]
> please make this little changes and PR to master so I can merge

Hey!
Thanks for the quick review!
I've added the requested changes.

**Comment by N4S4** [comment]
just as reminder pip package has been updated to latest version 0.1.3.1 including your functionality.

Thank you

**Pull request merged**